cctv/process_images.py

```diff
@@ -112,6 +126,56 @@ async def worker(
         await camera.sleep()
 
 
+async def update_camera_stack(app, cameras, session, boto_client):
+    """
+    Checks Knack to see if any cameras have been disabled or re-enabled by TPW staff.
+    Args:
+        app: knackpy app object
+        cameras: list of Camera objects
+        session (httpx.AsyncClient): The httpx session to use when fetching from cameras
+        boto_client (aiobotocore.session.AioSession): The (aio)boto3 session to upload images
+    """
+    fallback_img = load_fallback_img(FALLBACK_IMG_NAME)
+    await asyncio.sleep(random.uniform(0, INITIAL_MAX_RANDOM_SLEEP))
+    while True:
+        logger.debug("Checking for disabled cameras from Knack...")
+
+        try:
+            cameras_knack = get_camera_records(app, get_disabled=True)
+        except Exception as e:
+            logger.debug("Error trying to fetch camera data from Knack, skipping updating.")
+            await asyncio.sleep(SLEEP_SECONDS)
+            continue
```

Comment:
@johnclary added some error handling for when we get an error back from Knack. I wonder if this would suffice. Maybe there's another failure mode from the Knack API I'm not considering.

Reply:
i cant remember, does knack ever return a 200 that includes an error message?